core/local/change.js

```diff
@@ -172,7 +172,6 @@ function dirMoveFromUnlinkAdd (unlinkChange /*: LocalDirDeletion */, e /*: Local
     stats: e.stats,
     old: unlinkChange.old,
     ino: unlinkChange.ino,
-    overwrite: e.old,
```

Comment:
This is the key change.

Reply:
This is a revert of https://github.com/cozy-labs/cozy-desktop/pull/1181/files , no ?